<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTP_client_GUIClass</class>
 <widget class="QMainWindow" name="FTP_client_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP_client_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>850</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>661</width>
      <height>811</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>命令：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="send_button">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="param_lineEdit"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Port端口号：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="command_lineEdit"/>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="connect_button">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>连接</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IP地址：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>参数：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="IP_lineEdit"/>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="port_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>消息窗口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>655</width>
            <height>198</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="message_label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>10000</width>
               <height>10000</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件列表：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="files_list">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件类型</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件大小</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>创建时间</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>任务列表：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tasks_list">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件大小</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>任务类型</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>进度</string>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignBottom</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FTP_client_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
